src/utils/use-beacons.ts

```diff
@@ -0,0 +1,22 @@
+import {useIsBeaconsEnabled} from '@atb/beacons/use-is-beacons-enabled';
+import {KETTLE_API_KEY} from '@env';
+import {useCallback, useEffect} from 'react';
+import {NativeModules} from 'react-native';
+
+export const useBeacons = () => {
+  const [isBeaconsEnabled] = useIsBeaconsEnabled();
+  const isBeaconsSupported = isBeaconsEnabled && !!KETTLE_API_KEY;
+
+  const initializeBeacons = useCallback(async () => {
+    // This will set kettle if KETTLE_API_KEY is set on the native side
+    await NativeModules.KettleSDKExtension.initializeKettleSDK();
+  }, [NativeModules]);
+
+  useEffect(() => {
+    if (isBeaconsSupported) {
+      initializeBeacons();
+    }
+  }, [initializeBeacons, isBeaconsSupported]);
+
+  return {};
```

Comment:
Why return empty object?

Reply:
This is a template for the next PR https://github.com/AtB-AS/mittatb-app/blob/cf1c9575985c196f8145a38dc8978615b52eb672/src/utils/use-beacons.ts#L94